src/core/server/graph/schema/schema.graphql

```diff
@@ -7834,7 +7868,7 @@ type Mutation {
   """
   updateUserModerationScopes(
     input: UpdateUserModerationScopesInput!
-  ): UpdateUserModerationScopesPayload! @auth(roles: [ADMIN])
+  ): UpdateUserModerationScopesPayload! @auth(roles: [ADMIN, MODERATOR])
```

Comment:
Add `unscoped` to require moderators to be unscoped when using this mutation:

```suggestion
  ): UpdateUserModerationScopesPayload! @auth(roles: [ADMIN, MODERATOR], unscoped: true)
```

Reply:
This doesn't work. Admin's and org mods can also use this. They will not be scoped.